### ISSUE TITLE
[Android] ActivityUpdate: Keyboard focus order inside 'ActivityUpdate' is incorrect upon activation of 'Set Due date'.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6415]
### Target Platforms

Android

### SDK Version

Application Version: 2021.09.03.1 (38680868)

### Application Name

Adaptive Cards Android Visualizer

### Problem Description

[ADO 36142793](https://microsoft.visualstudio.com/OS/_workitems/edit/36142793)

**Repro Steps:**

- Launch the Adaptive cards app on Android.
- Navigate to 'Import' control and select it.
- Navigate to 'Sample cards' and select it.
- Navigate to available dropdown under 'Sample cards' and select it.
- Navigate all the controls under dropdown and select 'ActivityUpdate'.
- Navigate to 'Visual' tab control and select it.
- Navigate to 'Set Due Date' and activate it.

Observe the issue.

**Actual Result:**
Keyboard focus order inside 'ActivityUpdate' is incorrect upon activation of 'Set Due date'. Focus moves to 'Connect via QR Code' instead of 'Enter the due date' field.

**Expected Result:**
Keyboard focus order inside 'ActivityUpdate' should be correct upon activation of 'Set Due date'. Focus should move to 'Enter the due date' field.

### Card JSON

```JSON
ActivityUpdate.json
```
